--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -63,17 +63,17 @@
 SET (GATB_CORE_EXCLUDE_EXAMPLES  1)
 
 # GATB CORE
-include (GatbCore)
+find_package(gatbcore 1.3.0)
 
 ################################################################################
 # TOOL
 ################################################################################
 
 # We add the compilation options for the library
-add_definitions (${gatb-core-flags})
+add_definitions (${GATBCORE_DEFINITIONS})
 
 # We add the gatb-core include directory
-include_directories (${gatb-core-includes})
+include_directories (${GATBCORE_INCLUDE_DIRS})
 
 # We add the path for extra libraries
 link_directories (${gatb-core-extra-libraries-path})
@@ -84,14 +84,14 @@
 include_directories (${PROGRAM_SOURCE_DIR})
 file (GLOB_RECURSE  ProjectFiles  ${PROGRAM_SOURCE_DIR}/*)
 add_executable(${PROJECT_NAME} ${ProjectFiles})
-target_link_libraries(${PROJECT_NAME} ${gatb-core-libraries})
+target_link_libraries(${PROJECT_NAME} ${GATBCORE_LIBRARIES})
 
 # merci
 set (MERCI_SOURCE_DIR ${PROJECT_SOURCE_DIR}/merci)
 include_directories (${MERCI_SOURCE_DIR})
 file (GLOB_RECURSE  MerciFiles  ${MERCI_SOURCE_DIR}/*.cpp)
 add_executable("merci"  ${MerciFiles})
-target_link_libraries("merci" ${gatb-core-libraries})
+target_link_libraries("merci" ${GATBCORE_LIBRARIES})
 
 
 ################################################################################
