--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -131,8 +131,8 @@
 
 # We copy the project binary to the 'bin' directory
 INSTALL (TARGETS ${PROJECT_NAME} DESTINATION bin)
-INSTALL (DIRECTORY "${PROJECT_SOURCE_DIR}/test/" DESTINATION test)
-INSTALL (FILES LICENSE README.md DESTINATION bin/..)
+# INSTALL (DIRECTORY "${PROJECT_SOURCE_DIR}/test/" DESTINATION test)
+# INSTALL (FILES LICENSE README.md DESTINATION bin/..)
 
 # We include the "bin" tag into binary archive file name
 set (CPACK_PACKAGE_FILE_NAME  ${PROJECT_NAME}-${CPACK_PACKAGE_VERSION}-bin-${CMAKE_SYSTEM_NAME})
